components/card.css

```diff
@@ -0,0 +1,67 @@
+
+.card-vertical {
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+    width: 18rem;
+    height: auto;
+}
+.card-horizontal{
+    display: grid;
+    grid-template-columns: 1fr 3fr ;
+    grid-template-rows: 1fr auto;
+    width: 24rem;
+    height: auto;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+.card-img {
+    max-width: 11rem;
+    height: auto;
+}
+.card-details{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    overflow: hidden;
+}
+.sub-title{
+    color: var(--gray-dark);
+    font-size: 0.75rem;
```

Comment:
Try to stick to pixels for small values

Reply:
I wanted this ```font-size``` for ```sub-title``` class to be linked to root font size, which in this case is 16px.